File: PooCC02CAbel/Semana03/RelojJava/DisplayNumero.java
public class DisplayNumero {
    public int valor;
    public int limite;

    public DisplayNumero() {}

    public DisplayNumero(int valor, int limite) {
        this.valor = valor;
        this.limite = limite; 
    }

    public void incrementar() {
        if (valor < limite){
            valor += 1;
        } else {
            valor = 0;
        }
    }
}

File: PooCC02CAbel/Semana03/RelojPython/DisplayNumero.py
class DisplayNumero:
    valor = int
    limite = int
    def __init__(self, valor, limite):
        self.valor = valor
        self.limite = limite
    
    def incrementar(self):
        if self.valor < self.limite:
            self.valor += 1
        else:
            self.valor = 0
    
        